section4/exercises/burrito.rb

```diff
@@ -1,19 +1,36 @@
-# Add the following methods to this burrito class and 
+# Add the following methods to this burrito class and
 # call the methods below the class:
 # 1. add_topping
 # 2. remove_topping
 # 3. change_protein
 
 class Burrito
   attr_reader :protein, :base, :toppings
+
   def initialize(protein, base, toppings)
     @protein  = protein
     @base     = base
     @toppings = toppings
   end
+
+  def add_topping
```

Comment:
I'll give you a nudge for this one that may also help with the `change_protein` method - 

Currently, the `add_topping` method adds more of the original topping. Could you rework it so the user can choose a NEW topping to add to their burrito? How can information be added dynamically?

Reply:
Or would we be looking to change the attribute reader to accessor? Then do something like 
```ruby
def change_protein(protein)
@protein = protein
end
```

Reply:
I think that would work, and the same logic could be applied to the add_topping.